src/serve/javaEmulators.ts

```
@@ -167,12 +178,18 @@ export async function stop(targetName: JavaEmulators): Promise<void> {
 
 export async function start(targetName: JavaEmulators, args: any): Promise<void> {
   const emulator = EmulatorDetails[targetName];
-  const command = _getCommand(targetName, args);
-  if (!fs.existsSync(emulator.localPath)) {
-    utils.logWarning("Setup required, please run: firebase setup:emulators:" + emulator.name);
-    return Promise.reject("emulator not found");
+
+  const hasEmulator = fs.existsSync(emulator.localPath);
+  if (!hasEmulator) {
+    if (args.auto_download) {
+      await downloadEmulator(targetName);
+    } else {
+      utils.logWarning("Setup required, please run: firebase setup:emulators:" + emulator.name);
+      return Promise.reject("emulator not found");
```

Can this simply `throw` rather than returning a rejected `Promise`?